src/monotonic/monotonic.jl

```diff
@@ -9,11 +9,14 @@
         "Cardinal"              cubic cardinal splines, uses tension parameter which must be between [0,1]
                                 cubin cardinal splines can overshoot for non-monotonic data
                                 (increasing tension decreases overshoot)
+        "Akima"                 monotonic - tagents are dertermined at each given point locally,
+                                results are close to manual drawn curves
```

Comment:
What does "manually drawn curves" mean? Is that a quote from the article?

Reply:
Yes, the phrase "manually drawn cruves" comes indeed from a sentence in the article:

> Comparison indicates that the curve obtained by this new method is closer to a manually drawn curve than those drawn by other mathematical methods.

Reply:
So I think it's ok here, too. But I will change "manual" to "manually", of course!